DEPATISnet.js

```diff
@@ -198,16 +198,13 @@ function scrape(doc, url) {
 	});
 	
 	var pages = ZU.xpathText(doc, '//div[@id="inhalt"]/h2');
-	if (pages) {
-		pages = pages.match(/(Seiten|Pages):\s*([0-9]+)/)[2];
-		if (pages > 0) {
-			var pdfurl = "https://depatisnet.dpma.de/DepatisNet/depatisnet/" + pn + "_all_pages.pdf?window=1&space=menu&content=download_doc_verify&action=download_doc&docid=" + pn;
-			newItem.attachments.push({
-				title: "Fulltext",
-				url: pdfurl,
-				mimeType: "application/pdf"
-			});
-		}
+	if (pages && /(Seiten|Pages):\s*[0-9]+/.test(pages)) {
```

Comment:
This looks good to me. It would behave differently than the previous code in the case of `Seiten: 0` -- is that something we need to worry about? I'm a little puzzled by the old code if not for that possibility.

Reply:
Oh, that is right. Good catch! If we have Seiten: 0 then there is exactly no PDF available, e.g. https://depatisnet.dpma.de/DepatisNet/depatisnet?window=1&space=menu&content=treffer&action=bibdat&docid=AR000000047789A1 . But we can change the regex for that and maybe make a note to not forget.